### DISCUSSION TITLE
GitHub Copilot could not connect to serve

### DISCUSSION THREAD

**Discussion by HT-Moh** [community/community#45516] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am trying to make this extension work for days now but no luck, supposedly the extension should speed up development but at moment is doing the opposite, maybe I am doing something wrong!?
here is the error 
```
[ERROR] [default] [2023-01-27T10:24:37.999Z] GitHub Copilot could not connect to server. Extension activation failed: "command failed with error code 2: error parsing glob '!/{**/.git,**/.svn,**/.hg,**/CVS,**/.DS_Store,**/Thumbs.db,./.DS_Store,./.git,./.hg,./.svn,./${workspaceFolder}/**/.git,./${workspaceFolder}/**/.next,./${workspaceFolder}/**/.svn,./${workspaceFolder}/**/CVS,./${workspaceFolder}/**/node_modules,./${workspaceFolder}/**/out,./${workspaceFolder}/**/vendor,./CVS,./Thumbs.db,./.git/**,./node_modules/**,./.hg/**,./.svn/**}': nested alternate groups are not allowed
```

I tried to change `**/` to `./`, but didn't work, I appreciate your help.

**Comment by jge162** [comment]
This error message suggests that there is a problem with the glob pattern used in the extension's configuration.

reach out to support and submit a bug report to get help. https://support.github.com/contact/bug-report

or reach out to the extension's developer for help or consider using a different extension.

**Reply by SDAdham** [reply]
I have the same problem, any chance I can track the bug report that was created? Or do I need to create a new issue?

**Reply by jge162** [reply]
try using copilot nightly, should work.
